python/pyproject.toml

```diff
@@ -18,6 +18,6 @@
 [build-system]
 requires = [
     "setuptools>=42",
-    "wheel"
+    "wheel",
 ]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
```

Comment:
It doesn't look like this file substantially changed. Can we remove unnecessary changes here?